src/metrics/time-to-merge.ts

```diff
@@ -0,0 +1,48 @@
+import { Duration, Interval } from 'luxon';
+import { fetchMergedPullRequestNumbers } from '../utils/graphql-queries';
+import { loadPullRequest } from '../models/pull-request';
+import debug, { Debugger } from '../utils/debug';
+import durationToHuman from '../utils/duration-to-human';
+import { Metric, MetricData, percentiles } from '../metric';
+
+export default class TimeToMergeMetric implements Metric {
+  debug: Debugger;
+  data: MetricData[];
+  name = 'Pull Request Time-To-Merge';
+  constructor(public interval: Interval) {
```

Comment:
I think `public` is the default? 🤔

Reply:
This is a TS shorthand so that I don't have to set it. My understanding is these are equivalent (but the first is shorter):

```ts
class X {
  interval: Interval;
  constructor(public interval: Interval) {}
}
```

```ts
class X {
  interval: Interval;
  constructor(public interval: Interval) {
    this.interval = interval;
  }
}
```